src/utils/data.js

```diff
@@ -0,0 +1,41 @@
+import { DATA_TYPES, INCIDENT_FIELDS, INCIDENT_SEVERITY } from '../constants/Constants';
+
+function processIncidentData(data, fieldName) {
+  let processedData = data;
+  switch (fieldName) {
+    case INCIDENT_FIELDS.severity:
```

Comment:
shouldn't we handle the rest of the cases as well ?
![image](https://user-images.githubusercontent.com/61918543/120430698-11fdad80-c380-11eb-9095-23bc64ba4c07.png)